webcompat/templates/issue/issue-comment-submit.html

```diff
@@ -13,11 +13,12 @@
               <label class="wc-UploadForm-label wc-UploadForm-label--uploadForm">Attach images by <span class="wc-UploadForm-label--link">selecting them</span></label>
               <input accept=".jpe,.jpg,.jpeg,.gif,.png,.bmp" class="wc-ButtonUpload js-buttonUpload" id="image" name="image" type="file">
             </div>
-          </div>
+            <img class="wc-Upload-Loader wc-Issue-Loader js-Upload-Loader"
+                 src="/img/upload-loader.svg" alt="Uploading image">
+          </label>
```

Comment:
There's a bigger? issue. Nested labels. :)
https://www.w3.org/TR/html-markup/label.html


-     The label element must not contain any nested label elements.
-     The label element may contain at most one descendant input element, button element, select element, or textarea element.
-     The for attribute of the label element must refer to a form control.
-     The interactive element label must not appear as a descendant of the a element.
-     The interactive element label must not appear as a descendant of the button element.